src/hash/rescue/rpx/mod.rs

```diff
@@ -30,8 +30,8 @@ pub type CubicExtElement = CubeExtension<Felt>;
 /// - (M): `apply_mds` → `add_constants`.
 /// * Permutation: (FB) (E) (FB) (E) (FB) (E) (M).
 ///
-/// The above parameters target 128-bit security level. The digest consists of four field elements
-/// and it can be serialized into 32 bytes (256 bits).
+/// The above parameters target 128-bit security level for the permutation. The digest consists
+/// of four field elements and it can be serialized into 32 bytes (256 bits).
```

Comment:
Should we add a small section on padding and collision resistance somewhere here?

Reply:
Added a couple of sentences at the same location